src/libmongoc/src/mongoc/mongoc-client-pool.c

```diff
@@ -51,6 +51,7 @@ struct _mongoc_client_pool_t {
    int32_t error_api_version;
    bool error_api_set;
    mongoc_server_api_t *api;
+   bool client_initialized;
```

Comment:
to clarify, this is meant to be set when the first client created by this pool is initialized?

Reply:
Yes